### PR TITLE
Add some large model troubleshooting steps

### PR DESCRIPTION
**Pull request by damccorm**
This came up in a conversation with a user who had this issue

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

**Comment by damccorm** [comment]
R: @liferoad

File: website/www/site/content/en/documentation/ml/large-language-modeling.md [hunk 1]
```diff
@@ -122,3 +122,51 @@ A `ModelHandler` requires parameters like:
 * `device` – The device on which you wish to run the model. If device = GPU then a GPU device will be used if it is available. Otherwise, it will be CPU.
 * `inference_fn` -  The inference function to use during RunInference.
 * `large_model` - (see `Memory Management` above). Whether to use memory minimization techniques to lower the memory footprint of your model.
+
+### Troubleshooting Large Models
+
+#### Pickling errors
+
+When sharing a model across processes with `large_model=True` or using a custom model handler, Beam sends the input and output data across a process boundary.
+To do this, it uses a serialization method known as [pickling](https://docs.python.org/3/library/pickle.html).
```
Comment (liferoad) [inline website/www/site/content/en/documentation/ml/large-language-modeling.md:131]: shall we issue one warning about this pickling error when large_model=True is set? We could link this guide.

**Review by liferoad — APPROVED** [review]

File: website/www/site/content/en/documentation/ml/large-language-modeling.md (hunk 1 above)
Comment (damccorm) [reply website/www/site/content/en/documentation/ml/large-language-modeling.md:131]: That is a good idea - I'll follow up more on this when I'm back, either in this PR or a separate one

**Comment by damccorm** [comment]
I'll add this in a short follow up pr to make sure we're referencing the site correctly in the error (and since this PR is approved/ready to go)

**Comment by damccorm** [comment]
> I'll add this in a short follow up pr to make sure we're referencing the site correctly in the error (and since this PR is approved/ready to go)

https://github.com/apache/beam/pull/32063

**Pull request merged**